cli/test/commands/mint-to/index.test.ts

```diff
@@ -1,18 +1,22 @@
 import { expect, test } from "@oclif/test";
 import { initTestEnvIfNeeded } from "../../../src/utils/initTestEnv";
-import { getPayer, getSolanaRpcUrl } from "../../../src";
-import { Connection } from "@solana/web3.js";
+import {
+  defaultSolanaWalletKeypair,
+  getPayer,
+  getSolanaRpcUrl,
+} from "../../../src";
+import { Connection, Keypair } from "@solana/web3.js";
 import { createMint } from "@lightprotocol/compressed-token";
 import { confirmTx } from "@lightprotocol/stateless.js";
 import { requestAirdrop } from "../../helpers/helpers";
 
 describe("mint-to", () => {
   test.it(async () => {
     await initTestEnvIfNeeded();
-    await requestAirdrop(getPayer().publicKey);
-
+    const mintKeypair = defaultSolanaWalletKeypair() || Keypair.generate();
+    await requestAirdrop(mintKeypair.publicKey);
     const mintAmount = 100;
-    const mintAuthority = getPayer().publicKey.toBase58();
+    const mintAuthority = mintKeypair.publicKey.toBase58();
```

Comment:
same goes for this -  using the mintKeypair as authority is not intended but it works in this case because create_mint sets the mintkeypair as authority. we can add other testcases in follow up PRs